exchanges/bybit/bybit_test.go

```diff
@@ -3060,6 +3036,172 @@ func TestPushData(t *testing.T) {
 	}
 }
 
+func TestWSHandleAuthenticatedData(t *testing.T) {
+	t.Parallel()
+
+	b := new(Bybit) //nolint:govet // Intentional shadow to avoid future copy/paste mistakes
+	require.NoError(t, testexch.Setup(b), "Test instance Setup must not error")
+	b.API.AuthenticatedSupport = true
+	b.API.AuthenticatedWebsocketSupport = true
+	b.SetCredentials("test", "test", "", "", "", "")
+	testexch.FixtureToDataHandler(t, "testdata/wsAuth.json", func(ctx context.Context, r []byte) error {
+		if bytes.Contains(r, []byte("%s")) {
+			r = fmt.Appendf(nil, string(r), optionsTradablePair.String())
+		}
```

Comment:
🥃 🗻  This feels like a generic problem.
Wondering if we can text/template the content easily, and introduce a middleware:
```
FixtureToDataHandler(t, "foo.json", testexch.WrapFixture(b.wsHandleAuthenticatedData), testexch.FixtureVars{
    "optionsPair": optionsTradablePair,
}
```
and then in the fixture json:
```
{{ .vars.optionsPair.String }}
```

Reply:
Yeah that would be cool to do, can do this in another PR.